Makefile

```diff
@@ -196,8 +195,12 @@ clean:
 	rm -rf vendor
 	rm -f  $(DOCS_SYNOPISIS_DIR)/*.md
 
+.PHONY: clean_bindata ## Remove $(ADDON_BINDATA_DIR)
+clean_bindata:
+	rm -rf $(ADDON_BINDATA_DIR)
+
```

Comment:
This was added to address the root cause of https://github.com/minishift/minishift/pull/2282/